src/App.js

```diff
@@ -1,5 +1,134 @@
+import { Console, MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+  async play() {
+    let cars = await this.getCarName();
+    let times = await this.getCount();
+    Console.print("\n");
+    this.validCheckAboutCarName(cars);
+    this.validCheckAboutNumber(times);
+    let forwordArry = this.printEachProcess(cars, times);
+    let finerWiner = this.resultOfWinner(cars, forwordArry);
+    Console.print("최종 우승자 : " + finerWiner.join(', '));
+  }
+
+//---------------입력-------------------
+  async getCarName() {
+    const carsName = await MissionUtils.Console.readLineAsync('경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n');
+    let carNameArry = String(carsName).split(',');
+
+    return carNameArry;
+  }
+
+  async getCount() {
+    const howMove = await MissionUtils.Console.readLineAsync('시도할 횟수는 몇 회인가요?\n');
+    
+    return Number(howMove);
+  }
+
+//--------------횟수(숫자)유효성 판단--------------------
+  validCheckAboutNumber(howMove) {
+    if (Number.isNaN(howMove)) {
+      throw new Error(`[ERROR] 유효하지 않은 숫자 형식입니다.`);
+    }
+     const MIN_SIZE = 1;
+    if (howMove < MIN_SIZE) {
+
+      throw new Error(`[ERROR] 0보다 큰 값이어야 합니다.`);
+    }
+  }
+
+//---------------자동차 이름 유효성 판단-------------------
+  validCheckAboutCarName(carNameArry) {
+    if (this.checkDupl(carNameArry)) {
+      throw new Error(`[ERROR] 중복된 이름입니다.`);
+    }
+    if (this.checkNamelength(carNameArry)) {
+      throw new Error(`[ERROR] 유효하지 않은 이름 길이입니다.`);
+    }
+  }
+  
+  checkDupl(carNameArry) {
+    let carNameSet = new Set(carNameArry);
+    if (carNameArry.length !== carNameSet.size) {
+      return true;
+    }
+    return false;
+  }
+
+  checkNamelength(carNameArry) {
+    const NAME_MIN_LENG = 1;
+    const NAME_MAX_LENG = 5;
+    for (let i = 0; i < carNameArry.length; i++) {
```

Comment:
에어비앤비 스타일에서는 i++와 같은 Unary Operator의 사용을 권장하지 않고 있습니다!
대신 i += 1를 사용해 보는 것은 어떨까요?
아래 내용을 참고해 주시면 될 것 같아요:

[13.6](https://github.com/airbnb/javascript#variables--unary-increment-decrement) Avoid using unary increments and decrements (++, --). eslint [no-plusplus](https://eslint.org/docs/rules/no-plusplus)

> Why? Per the eslint documentation, unary increment and decrement statements are subject to automatic semicolon insertion and can cause silent errors with incrementing or decrementing values within an application. It is also more expressive to mutate your values with statements like num += 1 instead of num++ or num ++. Disallowing unary increment and decrement statements also prevents you from pre-incrementing/pre-decrementing values unintentionally which can also cause unexpected behavior in your programs.